src/bundle/Resources/public/js/scripts/embedded.item.actions.js

```diff
@@ -0,0 +1,308 @@
+        const locationInput = editEmbeddedItemForm.querySelector('[name="embedded_item_edit[location]"]');
+        const languageInput = editEmbeddedItemForm.querySelector(`[name="embedded_item_edit[language]"][value="${languageCode}"]`);
+
+        contentInfoInput.value = contentId;
+        locationInput.value = locationId;
+        languageInput.click();
+
+        editEmbeddedItemForm.submit();
+    };
+    const generateGoToActionItem = ({ contentId, locationId, productCode }) => {
+        const href = productCode
+            ? Routing.generate('ibexa.product_catalog.product.view', {
+                  productCode,
+                  languageCode: previewLanguageCode,
+              })
+            : Routing.generate('ibexa.content.translation.view', {
+                  contentId,
+                  locationId,
+                  languageCode: previewLanguageCode,
+              });
+
+        return {
+            label: Translator.trans(/*@Desc("Go to content")*/ 'embedded_items.action.go_to_label', {}, 'ibexa_content'),
+            href,
+        };
+    };
+    const generateEditActionItem = ({ contentId, locationId, productCode, languages }) => {
+        if (languages.length > 1) {
+            return {
+                label: Translator.trans(/*@Desc("Edit")*/ 'embedded_items.action.edit', {}, 'ibexa_content'),
+                branch: {
+                    groups: [
+                        {
+                            id: 'edit-group',
+                            items: languages.map(({ languageCode, name }) => {
+                                const languageEditAction = productCode
+                                    ? {
+                                          href: Routing.generate('ibexa.product_catalog.product.edit', {
+                                              productCode,
+                                              languageCode,
+                                          }),
+                                      }
+                                    : {
+                                          onClick: () => editContent({ contentId, locationId, languageCode }),
+                                      };
+
+                                return {
+                                    label: name,
+                                    ...languageEditAction,
+                                };
+                            }),
+                        },
+                    ],
+                },
+            };
+        }
+
+        const editAction = productCode
+            ? {
+                  href: Routing.generate('ibexa.product_catalog.product.edit', {
+                      productCode,
+                      languageCode: languages[0].languageCode,
+                  }),
+              }
+            : { onClick: () => editContent({ contentId, locationId, languageCode: languages[0].languageCode }) };
```

Comment:
```suggestion
            : { 
                  onClick: () => editContent({ 
                      contentId, 
                      locationId, 
                      languageCode: languages[0].languageCode,
                  }),
              };
```
wdyt? Looks more readable